pkg/vacuum/vacuum.go

```diff
@@ -215,10 +245,13 @@ func (e *Engine) worker(ctx context.Context, id int, todo <-chan string) {
 	for chunk := range todo {
 		log.Debug("msg", "vacuuming a chunk", "worker", id, "chunk", chunk)
 		sql := fmt.Sprintf(sqlVacuumFmt, chunk)
+		numberVacuumConnections.Inc()
 		_, err := e.pool.Exec(ctx, sql)
+		numberVacuumConnections.Dec()
 		if err != nil {
 			log.Error("msg", "failed to vacuum chunk", "chunk", chunk, "worker", id, "error", err)
 			// don't return error here. attempt to vacuum other chunks. keep working
 		}
+		tablesVacuumedTotal.Inc()
```

Comment:
We should only increment this in else statement (err == nil).

Reply:
May be we can have an error counter?

Reply:
good catch and good idea. thanks